Docs/API.md

```diff
@@ -301,6 +482,159 @@ catch(MinioException e)
 }
 ```
 
+### RemoveBucketAsync(RemoveBucketArgs args)
+
+`Task RemoveBucketAsync(RemoveBucketArgs args, CancellationToken cancellationToken = default(CancellationToken))`
+
+Removes a bucket.
+
+
+NOTE: -  removeBucket does not delete the objects inside the bucket. The objects need to be deleted using the removeObject API.
+
+
+__Parameters__
+
+ 
+|Param   | Type	  | Description  |
+|:--- |:--- |:--- |
+| ``args``  | _RemoveBucketArgs_  | RemoveBucketArgs Arguments Object which has bucket identifier information like bucket name  |
+| ``cancellationToken``| _System.Threading.CancellationToken_ | Optional parameter. Defaults to default(CancellationToken) |
+
+
+| Return Type	  | Exceptions	  |
+|:--- |:--- |
+|  Task  | Listed Exceptions: |
+|        | ``InvalidBucketNameException`` : upon invalid bucket name |
+|        | ``ConnectionException`` : upon connection error            |
+|        | ``AccessDeniedException`` : upon access denial            |
+|        | ``ErrorResponseException`` : upon unsuccessful execution            |
+|        | ``InternalClientException`` : upon internal library error        |
+|        | ``BucketNotFoundException`` : upon missing bucket          |
+
+
+__Example__
+
+
+```cs
+try
+{
+    // Check if my-bucket exists before removing it.
+    bool found = await minioClient.BucketExistsAsync(bktExistsArgs);
+    if (found)
+    {
+        // Remove bucket my-bucketname. This operation will succeed only if the bucket is empty.
+        await minioClient.RemoveBucketAsync(rmBktArgs);
+        Console.WriteLine(rmBktArgs.BucketName + " is removed successfully");
+    }
+    else
+    {
+        Console.WriteLine(bktExistsArgs.BucketName + " does not exist");
+    }
+}
+catch(MinioException e)
+{
+    Console.WriteLine("Error occurred: " + e);
+}
+```
+
+<a name="getVersioning"></a>
+### public async Task<VersioningConfiguration> GetVersioningAsync(GetVersioningArgs args)
+
+`Task<VersioningConfiguration> GetVersioningAsync(GetVersioningArgs args, CancellationToken cancellationToken = default(CancellationToken))`
+
+Get versioning information for a bucket.
+
+__Parameters__
+
+
+|Param   | Type	  | Description  |
+|:--- |:--- |:--- |
+| ``args``  | _GetVersioningArgs_  | GetVersioningArgs Arguments Object that has the info like bucket name, region. |
```

Comment:
| ``args``  | _GetVersioningArgs_  | GetVersioningArgs takes bucket name as argument |

Reply:
suggest remaining args description wording also be made concise.

Reply:
Concise wording included. Other stuff removed.